tests/test_plot.py

```diff
@@ -16,89 +17,42 @@ def setUp(self) -> None:
         ge.reset(False)
 
     def test_ndvi_histogram(self):
-        with UrllibMocker() as m:
-            m.post('http://mock-instance/anonymous', json={
-                "id": "c4983c3e-9b53-47ae-bda9-382223bd5081",
-                "project": None,
-                "view": None
-            })
+        # TODO: use `enterContext(cm)` instead of `with cm:` in Python 3.11
```

Comment:
this test does not finish within 10 minutes for me. Is there a way to capture the backend logs to see what's going on?

Reply:
`clear && pytest tests/test_plot.py --capture=no`

Reply:
>  2025-01-09T10:17:36.081650Z  INFO actix_server::server:192: Tokio runtime found; starting in existing Tokio runtime
Server started successfully

then nothing happens.

Reply:
Das nicht, aber trace

[3e1205b](https://github.com/geo-engine/geoengine-python/pull/211/commits/3e1205b36c78120179eb91c5b9c79025f02c3e51)